interfaces/Portalicious/src/app/pages/projects-overview/projects-overview.component.html

```diff
@@ -0,0 +1,55 @@
+<app-page-layout>
+  <div class="mb-6 flex items-end justify-between pt-6">
+    <h1
+      i18n
+      class="min-w-48 text-3xl"
+    >
+      All projects
+    </h1>
+
+    <div class="text-end">
+      <p-button
+        *ngIf="canCreateProjects"
```

Comment:
I could've wrapped the button in a `@if {}`-block; But I still like this pattern to "toggle" the button, based on some condition..

But feel free to rewrite for consistency. 👍